src/app/routes/cpsAsset/getInitialData/addBylineBlock/index.js

```diff
@@ -42,9 +41,13 @@ const addBylineBlock = originalJson => {
 
   const bylineBlock = getBylineBlock(json);
   if (bylineBlock) {
-    json.content.model.blocks = insertBlockAfterHeadline(bylineBlock, blocks);
+    if (
+      bylineBlock.model.blocks[0].name !== undefined ||
+      bylineBlock.model.blocks[0].title !== undefined
```

Comment:
This may read a bit better if you invert the logic so name and title have to be truthy (not an empty string, undefined or null)
`if(bylineBlock.model.blocks[0].name && bylineBlock.model.blocks[0].title)`